apis/nucleus/src/components/Header.jsx

```diff
@@ -71,9 +71,15 @@ const Header = ({ layout, sn, anchorEl, hovering }) => {
   const Toolbar = (
     <ActionsToolbar
       show={showToolbar}
-      selections={{ show: showInSelectionActions, api: sn.component.selections }}
+      selections={{
+        show: showInSelectionActions,
+        api: sn.component.selections,
+        onConfirm: focusHandler.refocusContent,
+        onCancel: focusHandler.refocusContent,
```

Comment:
Would this show when selecting with mouse as well?

Reply:
yes... It is not that pretty. I could try to make different key and click handlers. I think I ran into some problems when I tried before, but should be possible

Reply:
Would it be possible to check keyboard.active? Docs say that it is only active when space/enter is used to activate the chart

Reply:
yes but it is set to false when you blur the chart which you do when you navigate to the toolbar...

Reply:
Maybe we can check the event instead. Not super pretty, but don't refocus on click events or something